src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -211,18 +226,28 @@ def _to_list(_: bool, val: Any) -> List[Any]:
         ]
         calls = get_data_feed_call + get_current_values_call + get_timestampby_query_id_n_idx_call
         multi_call = Multicall(calls=calls, _w3=self.w3, require_success=require_success)
-        feed_details = await multi_call.coroutine()
-
-        return feed_details
+        try:
+            feed_details = await multi_call.coroutine()
+            return feed_details
+        except ValueError as e:
```

Comment:
Same thing re your logging msg here. Also, seems like it's repeating some logic above, so let's refactor this into a single function if possible.

Reply:
Same thing a couple times in the code further down